psl/psl-core/src/lib.rs

```diff
@@ -77,8 +120,8 @@ pub fn validate(file: SourceFile, connectors: ConnectorRegistry<'_>) -> Validate
 /// computationally or in terms of bundle size (e.g., for `query-engine-wasm`).
 pub fn parse_without_validation(file: SourceFile, connectors: ConnectorRegistry<'_>) -> ValidatedSchema {
     let mut diagnostics = Diagnostics::new();
-    let db = ParserDatabase::new(file, &mut diagnostics);
-    let configuration = validate_configuration(db.ast(), &mut diagnostics, connectors);
+    let db = ParserDatabase::new_single_file(file, &mut diagnostics);
+    let configuration = validate_configuration(db.ast_assert_single(), &mut diagnostics, connectors);
     let datasources = &configuration.datasources;
     let out = validate::parse_without_validation(db, datasources);
```

Comment:
`parse_without_validation` is the PSL entrypoint in `query-engine-wasm`.
If we plan on supporting multiple schema files @ Edge as well (and it seems we do), we should have an equivalent `parse_without_validation_multi_file` function definition.

Reply:
Plan for QE is to keep using single file for forceable future, so i thing we are ok with `parse_without_validation` accepting a single one for now.